process/block/shardblock.go

```diff
@@ -683,9 +697,11 @@ func (sp *shardProcessor) CreateBlock(
 func (sp *shardProcessor) createBlockBody(shardHdr *block.Header, haveTime func() bool) (data.BodyHandler, error) {
 	sp.blockSizeThrottler.ComputeMaxItems()
 
+	round := shardHdr.GetRound()
+
 	log.Debug("started creating block body",
 		"epoch", shardHdr.GetEpoch(),
-		"round", shardHdr.GetRound(),
+		"round", round,
```

Comment:
no need to get round out of shard hdr in a separate variable.

Reply:
yes. It was just for testing purposes to put a breakpoint on round